src/app/presenter/ChessBoardPresenter.ts

```diff
@@ -31,4 +35,34 @@ export class ChessBoardPresenter {
     const square: Square = { column: columns[position.x - 1], row: position.y as Row };
     return `${square.column.toLowerCase()}${square.row}`;
   }
+
+  private translatePositionToSquareNotation(position: Position): Square {
+    return { column: columns[position.x - 1], row: position.y as Row };
+  }
+
+  private translateSquareToAlgebraicNotation(square: Square): string {
+    return `${square.column.toLowerCase()}${square.row}`;
+  }
+
+  private checkEvents(eventsArray: (PieceWasMoved | PieceWasCaptured)[]): void {
+    eventsArray.forEach((event) => {
+      event.eventType === 'PieceWasMoved'
```

Comment:
To z pewnością nie jest dobry pomysł. Nie ma ograniczenia, że te eventy zawsze będą tylko 2 typów :) A zaraz nam też dojdą kolejne. Proponuje mieć obiekt w stylu: 
```.ts
{
'PieceWasMoved': (event: PieceWasMoved) => this.view.movePiece(....),
'PieceWasCaptured': (event: PieceWasCaptured) => this.view.capturePiece(...)
}
```
I wtedy na bazie typu eventu pobierasz funkcje i wykonujesz :)